src/main/java/com/study/gamereviewrest/exceptionHangler/ApiException.java

```diff
@@ -0,0 +1,20 @@
+package com.study.gamereviewrest.exceptionHangler;
+
+import lombok.Data;
+import org.springframework.http.HttpStatus;
+
+import java.time.ZonedDateTime;
+@Data
+public class ApiException {
```

Comment:
horrible package naming
should have been `com.venom.game.exceptions.GameException`
class should extend `Exception`

Reply:
I see that this is not an actual exception but a response pojo for in cases of exception
change the name to `ErrorResponse`
shouldn't contain throwable and httpStatus 
should be a string array for exception message and int for status

Reply:
timestamp should just be long